parachain/runtime/interlay/src/lib.rs

```diff
@@ -129,7 +129,9 @@ const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(10);
 /// by  Operational  extrinsics.
 const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 /// We allow for 2 seconds of compute with a 12 second average block time.
-const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND.saturating_mul(2 as u64);
+pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND
+    .saturating_div(2)
```

Comment:
We only allow 0.5 sec of block production?

Reply:
Yea that comment is wrong, see the configuration for statemine / statemint here: https://github.com/paritytech/cumulus/blob/3095325ac440cd3f30efb4acc0e6d7b84a1609ff/parachains/common/src/lib.rs#L94